src/main/java/org/kainos/ea/cli/Responsibility.java

```diff
@@ -0,0 +1,19 @@
+package org.kainos.ea.cli;
+
+public class Responsibility {
+    private String responsibilityName;
+
+    public Responsibility() {
+        this.responsibilityName = responsibilityName;
+    }
+
+    public String getResponsibilityName() {
+
```

Comment:
Does your linted not complain about these white spaces?

Reply:
@jkelso662 No it doesn't seem to mind about those whitespaces. I've removed them and I'll be more mindful in the future.